main/php_stdint.h

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
Will this change affect any PECL extensions?

Reply:
It might, but it shouldn't and the change is straight forward to use the standard C99 ``uint8_t`` type instead. But I will split this change into it's own PR and edit UPGRADING.INTERNAL.